agenta-backend/agenta_backend/models/api/organization_models.py

```diff
@@ -11,8 +12,14 @@ class TimestampModel(BaseModel):
 class Organization(TimestampModel):
     name: str
     description: Optional[str]
+    owner: User
+    members: Optional[List]
```

Comment:
List[str], no?

Reply:
Yes, you are right. 

I've corrected a bunch of these but haven't pushed them yet.

Reply:
Optional[List[UserDB]]

Reply:
Optional[List[ObjectId]]

Reply:
@mmabrouk Take a look.

Reply:
owner is fine as it is.